### DISCUSSION TITLE
Card component for peak

### DISCUSSION THREAD

**Discussion by rabol** [studio1902/statamic-peak#294] [Ideas]
Hi

Has anyone made a Card component for Peak that they want to share ?

Kind regards
Steen

**Comment by robdekort** [comment]
What are you looking for exactly?

**Reply by rabol** [reply]
something like this:
https://v1.tailwindcss.com/components/cards

**Reply by robdekort** [reply]
I often create a card component for when I loop collections for example. But this very much styling oriented and not so much logic/structure oriented so I'm not sure how this would fit in.

However check out the `clients`, `events` or `news` presets. The `components/_collection_handle_item.antlers.html` partials could be your cards depending on how you style them. Hope this helps?

**Reply by rabol** [reply]
I agree that it is very styling orientated.
but I think it must be possible to create a simple reusable card :)

I'll give it a shot

**Reply by robdekort** [reply]
Yeah for sure. Just make a partial out of it you can call in whenever.
